### ISSUE TITLE
Web browser 

### ISSUE THREAD

**Issue by JavaAppleJr** [ndless-nspire/Ndless#225]
I could find anywhere else to discuss this so I'm going to discuss it here. Since the Gossamer Web Browser was possible for Ti-84 is there a web browser for the TI-Nspire? It makes Sense since we have ndless and we have a touchpad (for mouse) and keyboard (for keyboard lol). Is there a way to wirelessly connect? Please tell me there is...

**Comment by lights0123** [comment]
@JavaAppleJr you can always run Lynx (or a full browser if you configure _a lot_ of swap) on Linux, and just use a USB to Ethernet or WiFi adapter.

**Comment by JavaAppleJr** [comment]
Is there a more full version? How do I use Lynx anyways?

**Comment by adriweb** [comment]
This has already been done several times / in several ways once you're into Linux. See for instance https://ivoah.net/blog/2016/03/20/how-to-install-debian-on-a-ti-nspire/  and more specifically  https://www.omnimaga.org/ti-nspire-projects/ti-nspire-cx-linux-adventures-(arch-linux-with-x11-internet-audio)/ and others you can find here and there.

As you can imagine, it's more useful as a technical exploit than actually doing anything...

On the CX II, if Linux runs on it one day, It could be better, speed wise.

**Comment by JavaAppleJr** [comment]
I have a version that is not supported for Ndless... so I can’t browse?

**Comment by lights0123** [comment]
@JavaAppleJr you can wait, or write something in Lua.

**Comment by Mordy24** [comment]
um i was just wondering if there was a ways to downgrade from os 4.5.2.8 to 4.5

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by System-Lag** [comment]
@Mordy24 not really, however I have come across one guy in reddit who claims to have ndless 4.5.0 running on his calculator with OS 4.5.2 and playing gbsp, but I’m not about to update my calculator to test it out. But you may as well give it a shot and see what happens

**Comment by jkcoxson** [comment]
Oh my goodness

**Comment by System-Lag** [comment]
@jkcoxson that’s my bad, I guess that’s  I get for using safari on an iPhone still running iOS 11

**Comment by jkcoxson** [comment]
Rip, lol.

**Comment by RedshiftVelocities** [comment]
@System-Lag Can you post a link to this reddit post? Thanks!

**Comment by Mordy24** [comment]
@System-Lag can you share the link

**Comment by RedshiftVelocities** [comment]
@Mordy24 generally speaking, if you want something from someone, it's best to be polite about it. It's also best not to spam them. Just so that you're aware.

**Comment by he77789** [comment]
> This has already been done several times / in several ways once you're into Linux. See for instance https://ivoah.net/blog/2016/03/20/how-to-install-debian-on-a-ti-nspire/ and more specifically https://www.omnimaga.org/ti-nspire-projects/ti-nspire-cx-linux-adventures-(arch-linux-with-x11-internet-audio)/ and others you can find here and there. As you can imagine, it's more useful as a technical exploit than actually doing anything... On the CX II, if Linux runs on it one day, It could be better, speed wise.

To whomever that comes across this issue later: Linux on the CX II is already possible, with USB support. Use files from https://tiplanet.org/nspire_linux_builds/ and use a dtb from https://tiplanet.org/forum/viewtopic.php?f=15&t=25228&start=10
